packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -161,6 +161,9 @@ class ElevatedButton extends ButtonStyleButton {
   /// [ButtonStyle.iconColor] and [iconSize] is used to construct
   /// [ButtonStyle.iconSize].
   ///
+  /// If [foregroundColor] is provided and [iconColor] is not provided, then the button icon will
```

Comment:
Maybe something like: If [iconColor] is null, the button icon will use [foregroundColor]. If [foregroundColor] is also null, the button icon will use the default icon color.